advanced_functionality/autogluon-tabular/container-training/Dockerfile.training

```diff
@@ -2,6 +2,7 @@ ARG REGISTRY_URI
 FROM ${REGISTRY_URI}
 
 RUN pip install autogluon
+RUN pip install shap
 RUN pip install PrettyTable
 RUN pip install bokeh
 
```

Comment:
It seems you need to install `smdebug` as well.

```
RUN pip install smdebug
```

Reply:
We are using a sagemaeker provided mxnet container image as a base image so the package is pre-installed.